tools/src/main/groovy/cz/o2/proxima/tools/groovy/internal/ClassloaderUtils.groovy

```diff
@@ -34,15 +34,16 @@ import org.codehaus.groovy.syntax.Reduction
 class ClassloaderUtils {
 
   static CompilerConfiguration createConfiguration() {
-    def ret = new CompilerConfiguration(CompilerConfiguration.DEFAULT)
+    CompilerConfiguration.DEFAULT
     /*
     ret = ret.addCompilationCustomizers(
         new ASTTransformationCustomizer(TypeChecked),
         new ASTTransformationCustomizer(CompileStatic))
      */
-    ret.setTargetBytecode("1.11")
+    /*
+    ret.setTargetBytecode("11")
     ret.setDebug(true)
-    return ret
+     */
```

Comment:
should we remove both commented code-blocks? if not, can you please document why?